installation/on_debian_and_ubuntu.md

```diff
@@ -6,37 +6,51 @@ In Debian derived distributions, you can use the official Crystal repository.
 
 First you have to add the repository to your APT configuration. For easy setup just run in your command line:
 
-```
-curl https://dist.crystal-lang.org/apt/setup.sh | sudo bash
+```bash
+curl -sSL https://dist.crystal-lang.org/apt/setup.sh | sudo bash
 ```
 
-That will add the signing key and the repository configuration. If you prefer to do it manually, execute the following commands as *root*:
+That will add the signing key and the repository configuration. If you prefer to do it manually, execute the following commands:
 
-```
-apt-key adv --keyserver keys.gnupg.net --recv-keys 09617FD37CC06B54
-echo "deb https://dist.crystal-lang.org/apt crystal main" > /etc/apt/sources.list.d/crystal.list
-apt-get update
+```bash
+sudo sh -c 'apt-key adv --keyserver keys.gnupg.net --recv-keys 09617FD37CC06B54 && \
+            echo "deb https://dist.crystal-lang.org/apt crystal main" > /etc/apt/sources.list.d/crystal.list && \
+            apt-get update'
 ```
 
+**NOTE:** There is a bug in Ubuntu 18.04 running on Windows Subystem for Linux, making `apt-key adv` fail.
+As a workaround, the first command can be replaced by `curl -sL "https://keybase.io/crystal/pgp_keys.asc" | sudo apt-key add`.
```

Comment:
Please replace this line in the instructions above and dont mention `adv` at all.

Reply:
Then `https://dist.crystal-lang.org/apt/setup.sh` should be changed accordingly.

Reply:
yes it should